### DISCUSSION TITLE
Link promotions with products model

### DISCUSSION THREAD

**Discussion by ollebergkvist** [swellstores/community#540] [Ideas]
Currently, displaying a promotional price for a product, like on an 'all products' page, requires two separate queries—one to the products endpoint and another to the promotions endpoint. This not only adds unnecessary complexity but also creates an extra network request. It would be beneficial to have an option to expand or include promotion prices directly in the products query.

**Comment by ericingram** [comment]
It's a good point. We are planning to add promotion pricing to the product API response. No ETA yet but will add your request to our notes.
